### ISSUE TITLE
Hangs on everything

### ISSUE THREAD

**Issue by 222464** [minerllabs/minerl#307]
I installed via pip, running the basic example leads to nothing happening, and I have to Ctrl + C.
The download function also doesn't work, it says "Server does not support HTTPRange. threads_count is set to 1." and then it also does nothing, no files are created in the directory I specified.

For the basic example, when I hit Ctrl + C it shows the following stack trace:

`Traceback (most recent call last):
  File "test.py", line 4, in <module>
    env = gym.make('MineRLNavigateDense-v0')
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/gym/envs/registration.py", line 145, in make
    return registry.make(id, **kwargs)
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/gym/envs/registration.py", line 90, in make
    env = spec.make(**kwargs)
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/gym/envs/registration.py", line 60, in make
    env = cls(**_kwargs)
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/minerl/env/core.py", line 133, in __init__
    self.instance = self._get_new_instance(port)
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/minerl/env/core.py", line 155, in _get_new_instance
    instance.launch()
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/minerl/env/malmo.py", line 463, in launch
    line = self.minecraft_process.stdout.readline().decode(mine_log_encoding)
KeyboardInterrupt
^CError in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/<username>/miniconda3/lib/python3.7/multiprocessing/popen_fork.py", line 28, in poll
    pid, sts = os.waitpid(self.pid, flag)
KeyboardInterrupt`

For the download (just importing and calling the download function), Ctrl + C yields the following stack trace:

`Traceback (most recent call last):
  File "download.py", line 3, in <module>
    minerl.data.download('data/')
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/minerl/data/download.py", line 92, in download
    obj.fetch_hash_sums()
  File "/home/<username>/miniconda3/lib/python3.7/site-packages/minerl/dependencies/pySmartDL/pySmartDL/pySmartDL.py", line 210, in fetch_hash_sums
    obj = urllib.request.urlopen(sums_url)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 525, in open
    response = self._open(req, data)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 543, in _open
    '_open', req)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 1362, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/home/<username>/miniconda3/lib/python3.7/urllib/request.py", line 1319, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 1026, in _send_output
    self.send(msg)
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 966, in send
    self.connect()
  File "/home/<username>/miniconda3/lib/python3.7/http/client.py", line 1422, in connect
    server_hostname=server_hostname)
  File "/home/<username>/miniconda3/lib/python3.7/ssl.py", line 423, in wrap_socket
    session=session
  File "/home/<username>/miniconda3/lib/python3.7/ssl.py", line 870, in _create
    self.do_handshake()
  File "/home/<username>/miniconda3/lib/python3.7/ssl.py", line 1139, in do_handshake
    self._sslobj.do_handshake()
KeyboardInterrupt
`

OS: Manjaro Linux

This is using miniconda.

**Comment by act65** [comment]
I have the same issue.

- Ubuntu 16.04.6
- Conda / Python 3.7

**Comment by Miffyli** [comment]
On running MineRL: Make sure you have correct java versions installed etc. First time running it will take long time as it compiles some Java stuff for MineRL/Malmo to work.

As for download: The server links in current `master` are out of date (I believe) and do not work. @MadcowD said the new data and MineRL version will be up soon, hopefully before the MineRL2020 competition starts :)

**Comment by 222464** [comment]
I got it to work! The environment running, at least. It seems if you just let it sit there for a while, a more helpful (existent) error message appears. It seems it wasn't able to find tools.jar, which means I have the jre and not jdk (despite the thing I installed having "JDK" in the name). I couldn't find a version 8 in the official Manjaro (Arch) repositories, so I went to AUR and found:

> https://aur.archlinux.org/packages/jdk8

Which worked for me.

The download doesn't work still, but I guess I have to wait on that for now if the server links are just out of date.
